### ISSUE TITLE
Reviewer 2 comments for JOSS submission

### ISSUE THREAD

**Issue by marouenbg** [yaml2sbml-dev/yaml2sbml#127]
yaml2sbml's implementation was executed flawlessly with a complete set of docs and several examples.
These are my comments in relation the review process https://github.com/openjournals/joss-reviews/issues/3215

# Comments on the paper:
These are a few suggestions for the authors about the main text.

- Currently 9 software toolboxes support PEtab as an input format, among them COPASI, d2d,
Would it be possible to complete the list?
- Due to the aforementioned tools, model simulation or parameter estimation has become a matter of a few lines of code or clicks
I suggest using Thanks to the aforementioned tools

- Hence ODE model definition is a relevant bottleneck,
I would suggest: However, ODE model definition is often a bottleneck

- Python-based model editor that allows one to generate
My suggestion: Python-based model editor that allows to generate

- Figure 1 gives an overview over of the typical
 Figure 1 gives an overview of the typical
 
 - Building the input format on YAML allows one to easily parse and validate the model
 Building the input format on YAML allows to parse and validate the model easily

**Comment by marouenbg** [comment]
# Comments on the tool

- specify the dependencies in the first tutorial - finite state projection ( amici package) 
- the URL links in the beginning of Format features tutorial are not working

**Comment by jvanhoefer** [comment]
Dear Marouen, 

Thank you for your helpful comments and for reviewing our submission! I addressed your suggestions in two PRs for your comments regarding the tool (#128) and the text (#129).  I agree, that your reformulation will improve the manuscript! Regarding

> * Currently 9 software toolboxes support PEtab as an input format, among them COPASI, d2d,
Would it be possible to complete the list?

I added a link to the PEtab documentation, that gives a dynamically updated list with detailed overview over tools + support (as there are to my knowledge more tools who will have a PEtab support soon).
